docs/SPECIFICATION.md

```diff
@@ -424,6 +427,49 @@ A LinkRemove in a message `m` is valid only if it passes these validations:
 
 1.  `m.data.type` must be `MESSAGE_TYPE_LINK_REMOVE`
 
+## 1.7 Username Proof
+
+```protobuf
+enum UserNameType {
+  USERNAME_TYPE_NONE = 0;
+  USERNAME_TYPE_ENS_FNAME = 1;
```

Comment:
This is just called `USERNAME_TYPE_FNAME`

Reply:
should we rename this to ENS_FNAME in the implementation? 

the rationale is that this makes it clear that the fname is a type of ENS name, not something distinct

Reply:
Sure, will rename.